apps/vessel-history/types/index.ts

```diff
@@ -122,9 +123,24 @@ export interface RiskOutput {
   highestRisk: RiskLevel
 }
 
+export enum IuuStatus {
+  notListed = 0,
+  previouslyListed = 1,
+  currentlyListed = 2,
+}
+
+export interface Iuu {
+  source: string;
+  startDate: string;
+  endDate?: string;
+  originalStartDate: number;
+  originalEndDate?: number;
+}
+
 export interface VesselWithHistory extends Vessel {
   history: VesselFieldsHistory
   iuuStatus?: number
+  iuuListing?: Iuu
```

Comment:
This field should have the same type that a single item of the `iuuListing.byDate` inside the `history` field, this is `ValueItem<string>`